### PR TITLE
Make Originating Identity feature gate on by default

### PR DESCRIPTION
**Pull request by jboyd01**
Enable Originating Identity Feature Gate by default.

Also add Feature Gate doc, using [K8s FG doc](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) as a template

doc preview available here:  https://deploy-preview-2277--svc-cat.netlify.com/docs/devguide/#featuregates

### CODE REVIEW

**Comment by carolynvs** [comment]
@jboyd01 Since you've done a bit of research of how "big k8s" does flags, do you recommend that we make feature flags as `FeatureXDisabled` going forward? So that we don't have to change the name like we did here?

**Comment by jboyd01** [comment]
@carolynvs the difficulty (at least my perceived difficulty) is with helm and the ability to check if a variable is set to false vs not defined.  I'd keep the FeatureGate as `FeatureXEnabled`, but the Helm variable should be featureXDisabled = true in variables.yaml and eventually that transitions to false.  The associated logic in the charts may seem backwards at first, but in the long run I think it sets us up properly to transition the feature to be enabled.   If we get agreement on this I'll update the dev guide doc.

**Comment by carolynvs** [comment]
/hold

Adding a hold while Jay tests out our working theory that you can't have undefined values in a helm chart. Let me know what you find!

**Comment by jboyd01** [comment]
@carolynvs I've rehacked the charts as we discussed, this looks like a good approach, and I doc'd it.  Bottom line is that within helm, we expect you to still set featureFooEnabled=true within values.yaml and within the template, I changed to be if not featureFooEnabled then set the fg=false.   Works and seems correct to me.

File: docs/devguide.md [hunk 1]
```diff
@@ -317,6 +317,41 @@ Counterfeiter by running `go get github.com/maxbrunsfeld/counterfeiter`.
...
+```
+{% endraw %}
+
+When the feature has had enough testing and the community agrees to change the
+default to true, update `features.go` to set the default to true, update the
+`values.yaml` changing the default for feature foo to `true` and change the
+template logic like this:
+{% raw %}
+```yaml
+    {{- if not .Values.fooEnabled }}
```
[the remark this answers is not in the document]
Comment (jboyd01) [reply docs/devguide.md]: absolutely, that's pretty obvious.  Will simplify doc and charts.

**Comment by jboyd01** [comment]
Fowled my last commit :(
/hold

**Comment by jboyd01** [comment]
all set.  Too much flux for such a simple thing.  The end result is simple though, thanks @carolynvs

**Comment by jberkhahn** [comment]
/approved

nice docs

**Pull request merged**